app/services/vcard/generator.rb

```diff
@@ -0,0 +1,33 @@
+module Vcard
+  class Generator
+    VERSION = '3.0'
+    attr_reader :person, :vcard
+    def self.call(person)
+      new(person).call
+    end
+
+    def initialize(person)
+      @person = person
+    end
+
+    def call
+      @vcard = VCardigan.create(version: VERSION)
```

Comment:
is this a typo? its declared `@vcard` here but referenced as `vcard` in the rest of the method

Reply:
sorry, I just saw the `attr_reader :vcard` declaration. But I am not so sure about the value of an instance variable that is only ever used in the same invocation of the method that sets it?